src/main/java/com/github/kaklakariada/fritzbox/CmdLine.java

```diff
@@ -0,0 +1,315 @@
+/**
+ * A Java API for managing FritzBox HomeAutomation
+ * Copyright (C) 2017 Christoph Pirkl <christoph at users.sourceforge.net>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.github.kaklakariada.fritzbox;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.kohsuke.args4j.CmdLineException;
+import org.kohsuke.args4j.CmdLineParser;
+import org.kohsuke.args4j.Option;
+import org.kohsuke.args4j.spi.StringArrayOptionHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.github.kaklakariada.fritzbox.model.homeautomation.Device;
+import com.github.kaklakariada.fritzbox.model.homeautomation.DeviceList;
+
+/**
+ * command line interface for home automation
+ * 
+ * @author wf
+ *
+ */
+public class CmdLine {
+  private final static Logger LOG = LoggerFactory.getLogger(CmdLine.class);
+
+  public static final String VERSION = "0.4.2";
```

Comment:
Please don't hard code version. Maybe read it from a file on classpath added during the build.